For clock_gettime() we have similar issues as we have for
getentropy(); the initial fix is quite rude: disable it.

diff --git a/m4/check-libc.m4 b/m4/check-libc.m4
index f2eb3eb..369561f 100644
--- a/m4/check-libc.m4
+++ b/m4/check-libc.m4
@@ -64,8 +109,6 @@ AM_CONDITIONAL([HAVE_ARC4RANDOM_BUF],
 
 # Check for getentropy fallback dependencies
 AC_CHECK_FUNC([getauxval])
-AC_SEARCH_LIBS([clock_gettime],[rt posix4])
-AC_CHECK_FUNC([clock_gettime])
 AC_SEARCH_LIBS([dl_iterate_phdr],[dl])
 AC_CHECK_FUNC([dl_iterate_phdr])
 ])
